main.tf

```diff
@@ -12,6 +12,11 @@ resource "aws_ram_principal_association" "this" {
 
   principal          = var.principal
   resource_share_arn = var.resource_share_arn
+
+  # The invitation sometime takes a few seconds to propagate
+  provisioner "local-exec" {
+    command = "python -c 'import time; time.sleep(10)'"
```

Comment:
I believe this was just at 5 seconds before, and I don't remember problems with that length...

Reply:
It was 3, actually...

Reply:
Fixed